packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -156,20 +156,14 @@ export const useCCTPDeposits = ({
   pageSize,
   enabled
 }: fetchCctpParams) => {
-  const { data, error, isLoading } = useSWRImmutable(
+  const { data, error, isLoading, mutate } = useSWRImmutable(
```

Comment:
We use `useSWRImmutable` so it doesn't revalidate https://swr.vercel.app/docs/revalidation.en-US#disable-automatic-revalidations